src/interceptorFilter.factory.js

```diff
@@ -0,0 +1,71 @@
+/* global angular */
```

Comment:
This is currently inside bearhug, but I think it should be a separate project/module.

It's very simple: it is a wrapper for an HTTP interceptor that is used to selectively apply that interceptor.

For example, we may care to only apply the interceptor to a particular endpoint, or to non-HTML files.  `interceptorFilter` allows you to separate the predicates for describing when an interceptor applies from the interceptor itself.